Documentation/api.md

```diff
@@ -1,5 +1,27 @@
 # etcd API
 
+- [Key Space Operations](#key-space-operations)
```

Comment:
i think the table is too huge.

how about combine some of them? for example, list dir/delete dir -> dir opeartion